FFmpegConfig.sh

```diff
@@ -27,6 +27,17 @@ configureArgs="\
     --enable-${sharedOrStatic} \
     --enable-cross-compile \
     --enable-debug \
+    --disable-encoders \
+    --disable-devices \
+    --disable-hwaccels \
+    --disable-programs \
+    --disable-d3d11va \
+    --disable-dxva2 \
```

Comment:
These should only be disabled for UWP build:

    --disable-encoders \
    --disable-devices \
    --disable-hwaccels \
    --disable-programs \
    --disable-d3d11va \
    --disable-dxva2 \

The new Win32 build is a fully featured FFmpeg.exe, so it should contain encoders, programs and all other stuff.

By the way, I think we could completely remove these:

    --disable-d3d11va \
    --disable-dxva2 \

They are both hwaccels, and we already have disable-hwaccels. So these are redundand.